.gitignore

```diff
@@ -8,4 +8,6 @@
 /captures
 .externalNativeBuild
 /.idea
-.idea
\ No newline at end of file
+.idea
+/app/src/main/res/audio
```

Comment:
i think we need to remove the / in front of /app so that this is just app/src/main/res/audio becuase the leading slash indicates root folder, and we need releative folder.